Makefile

```diff
@@ -100,7 +100,15 @@ bench:
 	./scripts/clean_fcctr.sh
```

Comment:
line 23: you need to add `profile` to SUBDIRS. Unit tests broke

Reply:
Done